s3/s3_test.go

```diff
@@ -71,6 +71,11 @@ func TestS3(t *testing.T) {
 				So(err, ShouldBeNil)
 				So(string(buf), ShouldEqual, testData)
 			})
+
+			Convey("And remove it", func() {
+				err := s3.RemoveFile(basename)
+				So(err, ShouldBeNil)
```

Comment:
This needs an actual test that it did something. Eg. OpenFile() should fail.

Reply:
Added check before to confirm file exists, and check after to confirm it has been removed.